src/deluge/processing/audio_output.cpp

```diff
@@ -394,7 +394,7 @@ bool AudioOutput::wantsToBeginArrangementRecording() {
 
 bool AudioOutput::setActiveClip(ModelStackWithTimelineCounter* modelStack, PgmChangeSend maySendMIDIPGMs) {
 	if (activeClip
-	    && (activeClip != modelStack->getTimelineCounter()
+	    && (!modelStack || activeClip != modelStack->getTimelineCounter()
```

Comment:
should this be getActiveClip()?

Reply:
Nah this is an output, it owns the activeClip